nucypher/blockchain/eth/signers.py

```diff
@@ -251,102 +258,165 @@ def unlock_account(self, account: str, password: str, duration: int = None) -> b
         return True
 
     @validate_checksum_address
-    def lock_account(self, account: str):
+    def lock_account(self, account: str) -> bool:
         return True
 
 
-class KeyStoreSigner(Signer):
+class KeystoreSigner(Signer):
     """Local Web3 signer implementation supporting a single account/keystore file"""
 
-    IS_SIGNER_LOCAL = True
+    __keys: Dict[str, dict]
+    __signers: Dict[str, LocalAccount]
 
-    __key: Dict
-    __account: str
-    __signer: LocalAccount
+    class InvalidKeyfile(ValueError):
```

Comment:
Should `InvalidKeyfile` subclass `SignerError`? I'm not confident in that, so thought I would throw it out there to get your thoughts 😊

Reply:
Yea... Perhaps better to be decisive on this - I can see it either way, `ValueError` is totally acceptable here, IMO.

Is `SignerError(ValueError)`?

Reply:
I could see `SignerError(ValueError)` instead of `Exception`